skyrim-platform/src/tilted/ui/TextToDraw.h

```diff
@@ -1,11 +1,12 @@
 #pragma once
+
 #include <array>
 #include <string>
 
 struct TextToDraw
 {
   float x = 0.f;
   float y = 0.f;
-  std::wstring kString;
-  std::array<float, 4> color = { 0.f, 0.f, 1.f, 1.f };
+  std::wstring string;
+  std::array<float, 4> color = { 255.f, 255.f, 255.f, 1.f };
```

Comment:
As far as I know values here are in range 0..1, not 0..255
so 255 isn't meaningful here